modules/dag-shared/src/main/scala/org/tessellation/dag/snapshot/StateChannelSnapshotBinary.scala

```diff
@@ -14,10 +15,12 @@ case class StateChannelSnapshotBinary(
   lastSnapshotHash: Hash,
   content: Array[Byte]
 )
-
 object StateChannelSnapshotBinary {
   implicit val show: Show[StateChannelSnapshotBinary] = s => s"StateChannelSnapshotBinary(lastSnapshotHash=${s.lastSnapshotHash.show})"
 
+  implicit val arrayByteOrder: Order[Array[Byte]] = Order.by(_.toSeq)
```

Comment:
Please create a generic order for `Array[A]` and add it to the `OrhanInstances`

Reply:
:+1: